.github/workflows/test_linux_job.yml

```diff
@@ -66,24 +66,15 @@ jobs:
       gpu-arch-version: "12.1"
       timeout: 60
       script: |
-        conda create --yes --quiet -n test python=3.9
-        conda activate test
+        /opt/python/cp39-cp39/bin/python -m venv .venv
+        source .venv/bin/activate
         python3 -m pip install --index-url https://download.pytorch.org/whl/nightly/cu121 --pre torch
         # Can import pytorch, cuda is available
         python3 -c 'import torch;cuda_avail = torch.cuda.is_available();print("CUDA available: " + str(cuda_avail));assert(cuda_avail)'
         python3 -c 'import torch;t = torch.ones([2,2], device="cuda:0");print(t);print("tensor device:" + str(t.device))'
         nvidia-smi
         nvcc --version | grep "cuda_12.1"
         [[ "${CUDA_HOME}" == "/usr/local/cuda-12.1" ]] || exit 1
-  test-docker-image:
```

Comment:
Not sure if random docker image is needed here. I guess I can use manylinux_2_28 from https://github.com/pypa/manylinux here